src/channels/router_channel_utils.erl

```diff
@@ -155,49 +155,49 @@ url_test_() ->
     [
         ?_assertEqual(
             {
-                <<"https://website.com/">>,
+                <<"https://website.com">>,
                 []
             },
             make_url("https://website.com", [], #{})
         ),
         ?_assertEqual(
             {
-                <<"https://website.com/?one=two">>,
+                <<"https://website.com?one=two">>,
```

Comment:
Removing the slash makes the string no longer be a valid URL.

If this is truly intentional, please add a comment noting the fact.

This applies to multiple instances, below this line.

Reply:
I guess not anymore?
https://stackoverflow.com/a/42193734/1921234

Reply:
So many RFCs, so little time...